common/metrics/metric_defs.go

```diff
@@ -694,6 +694,10 @@ const (
 	PersistenceRangeCompleteVisibilityTasksScope = "RangeCompleteVisibilityTasks"
 	// PersistenceGetReplicationTaskScope tracks GetReplicationTask calls made by service to persistence layer
 	PersistenceGetReplicationTaskScope = "GetReplicationTask"
+	// PersistenceGetArchivalTaskScope tracks GetArchivalTask calls made by service to persistence layer
+	PersistenceGetArchivalTaskScope = "GetArchivalTask"
+	// PersistenceGetArchivalTasksScope tracks GetArchivalTasks calls made by service to persistence layer
+	PersistenceGetArchivalTasksScope = "GetArchivalTasks"
```

Comment:
Plz add scopes for complete and range complete api as well.

Reply:
Done